Makefile

```diff
@@ -145,6 +155,26 @@ ifeq ($(TARGET_PLATFORM),linux)
 		-l LICENSE \
 		-o $@
 endif
+ifeq ($(TARGET_PLATFORM),win32)
+	./scripts/build/electron-configure-package-win32.sh -p $(word 2,$^) -a $< \
+		-n "$(APPLICATION_NAME)" \
+		-d "$(APPLICATION_DESCRIPTION)" \
+		-v "$(APPLICATION_VERSION)" \
+		-l LICENSE \
+		-r "$(APPLICATION_COPYRIGHT)" \
+		-c "$(COMPANY_NAME)" \
```

Comment:
...and (again, for consistency) could you change `-r "$(APPLICATION_COPYRIGHT)"` to `-c "$(APPLICATION_COPYRIGHT)"` ? :smile: